data/systemd/snapd.mounts-pre.target

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Mounting snaps
+RefuseManualStart=yes
+After=zfs-mount.service
+After=ostree-remount.service
+# Started from snapd.mounts-pre.service
+# X-Snapd-Snap: do-not-start
```

Comment:
Is this a leftover?

Reply:
There is a mistake, it is supposed to say `# Started from snapd.mounts.target`. I will fix that.

This is intended. Otherwise snapd tries to start it, when it is systemd that will start them through wants in the units.